readme.md

```diff
@@ -81,15 +83,15 @@ Once installation is working, take a look at the existing code to make sure you
 * **Question:** This code is currently _very_ insecure. Why?
 
 * **Your Answer:**
-
+the password sent in plaintext and stored in plaintext
 * **Question:** What would happen if three different users tried to sign up with the same username? How can we prevent that?
 
 * **Your Answer:**
-
```

Comment:
Yes do a check first, currently all three users would be allowed to create accounts with identical usernames.